docs/payment-methods/FRENCH_MEAL_VOUCHER.md

```diff
@@ -0,0 +1,145 @@
+
+
+```Groovy
+implementation "com.adyen.checkout:meal-voucher-fr:YOUR_VERSION"
+```
+
+To [launch and show the Component](https://docs.adyen.com/online-payments/build-your-integration/sessions-flow?platform=Android&integration=Components#launch-and-show) use `MealVoucherFRComponent`.
+```Kotlin
+val component = MealVoucherFRComponent.PROVIDER.get(
+    activity = activity,
+    checkoutSession = checkoutSession,
+    paymentMethod = paymentMethod,
+    configuration = checkoutConfiguration,
+    componentCallback = callback,
+)
+```
+
+The `componentCallback` which is passed to the `MealVoucherFRComponent.PROVIDER` requires the following methods to be implemented:
+
+| Method                                                            | Explanation                                                                                                                                                                                                                                                                                                      |
+|-------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `onRequestOrder()`                                                | In this method you should make a network call to the [/orders](https://docs.adyen.com/api-explorer/Checkout/latest/post/orders) endpoint of the Checkout API through your server. This method is called when the user is trying to pay a part of the amount using a partial payment method.                      |
+| `onBalanceCheck(paymentComponentState: PaymentComponentState<*>)` | In this method you should make a network call to the [/paymentMethods/balance](https://docs.adyen.com/api-explorer/Checkout/latest/post/paymentMethods/balance) endpoint of the Checkout API through your server. This method is called right after the user enters their meal voucher details and submits them. |
+
+The following example shows how to implement the `MealVoucherFRComponentCallback`:
+
+```Kotlin
+override fun onRequestOrder() {
+    // Make a POST /orders request
+    if (isSuccessfulResponse()) {
+        val orderResponse = OrderResponse.SERIALIZER.deserialize(jsonResponse)
+        frenchMealVoucherComponent.resolveOrderResponse(event.order)
+    }
+}
+
+override fun onBalanceCheck(paymentComponentState: PaymentComponentState<*>) {
+    // Make a POST /paymentMethods/balance request
+    if (isSuccessfulResponse()) {
+        val balanceResult = BalanceResult.SERIALIZER.deserialize(jsonResponse)
+        frenchMealVoucherComponent.resolveBalanceResult(event.balanceResult)
+    }
+}
+```
+
+### Configurations
```

Comment:
This section is duplicated, maybe we can just have it as just one section at the bottom, regardless of drop-in or components

Reply:
The intro links to two separate pages in docs, one for Drop-in and another one for Components. I will move the duplicated part out and we can check how it looks.